hw/ip/csrng/data/csrng.hjson

```diff
@@ -86,9 +86,8 @@
   registers: [
     { name: "REGWEN",
       desc: "Register write enable for all control registers",
-      swaccess: "ro", // lock is managed by HW
-      hwaccess: "hwo",
-      hwext: "true",
+      swaccess: "rw0c",
+      hwaccess: "hro",
```

Comment:
@mwbranstad i think you can mark `hwaccess` as none, then you won't need to do the `unused_q`.

Reply:
nice! always found that irritating, so thanks for the tip.